api/src/data_inclusion/api/app.py

```diff
@@ -71,6 +72,11 @@ def create_app() -> fastapi.FastAPI:
 
     if settings.ENV == "dev":
         setup_debug_toolbar_middleware(app)
+    elif settings.ENV in ["prod", "staging"]:
```

Comment:
preferer un bloc `if` par code de config, car là c'est pas très "ouvert à l'extension" (S**O**LID) e.g. j'ajoute un bout de conf pour `prod` uniquement, je dois modifier le code existant

```suggestion
    if settings.ENV in ["prod", "staging"]:
```